lumigator/python/mzai/jobs/evaluator_lite/evaluator_lite/eval_metrics.py

```diff
@@ -0,0 +1,82 @@
+import evaluate
+import numpy as np
+from loguru import logger
+
+
+class EvaluationMetrics:
+    def __init__(self, metrics):
+        self._supported_metrics = {
+            "rouge": self._rouge,
+            "meteor": self._meteor,
+            "bertscore": self._bertscore,
+        }
+
+        # chosen metrics are the intersection between the provided and the supported ones
+        self._chosen_metrics = set(metrics) & set(self._supported_metrics.keys())
+        # unsupported metrics are the difference between the provided and the supporterd ones
+        self._unsupported_metrics = set(metrics) - set(self._supported_metrics.keys())
+
+        if len(self._chosen_metrics) == 0:
+            logger.info("No valid metrics selected")
+        else:
+            logger.info(f"Chosen metrics: {self._chosen_metrics}")
+
+        if len(self._unsupported_metrics) > 0:
+            logger.info(f"Unsupported metrics: {self._unsupported_metrics}")
```

Comment:
I suggest turning this into a `warning`. This is something that the user should pay attention to: "You have selected to run an evaluation job with a few unsupported metrics. I'm removing these and continue only with the supported ones."